source/file.c

```diff
@@ -41,9 +41,10 @@ int aws_byte_buf_init_from_file(struct aws_byte_buf *out_buf, struct aws_allocat
 
     if (fp) {
         if (fseek(fp, 0L, SEEK_END)) {
-            AWS_LOGF_ERROR(AWS_LS_COMMON_IO, "static: Failed to seek file %s with errno %d", filename, errno);
+            int errno_value = errno;
```

Comment:
trivial: annoying, but explain the subtle bug for any future programmers who come in here. I copy/pasted this everywhere when I did the windows GetLastError() pass
```suggestion
            int errno_value = errno; /* logging may reset error, so cache it */
```